### PR TITLE
Add missing db prefixes

### PR DESCRIPTION
**Pull request by Gomez**

### CODE REVIEW

**Comment by ChristophWurst** [comment]
:+1: nice

File: lib/db/aliasmapper.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function find($aliasId, $currentUserId) {
 	 * @return Alias[]
 	 */
 	public function findAll($accountId, $currentUserId) {
-		$sql = 'select *PREFIX*mail_aliases.* from *PREFIX*mail_aliases join *PREFIX*mail_accounts on oc_mail_aliases.account_id = oc_mail_accounts.id where *PREFIX*mail_accounts.user_id = ? AND *PREFIX*mail_aliases.account_id=?';
+		$sql = 'select *PREFIX*mail_aliases.* from *PREFIX*mail_aliases join *PREFIX*mail_accounts on *PREFIX*mail_aliases.account_id = oc_mail_accounts.id where *PREFIX*mail_accounts.user_id = ? AND *PREFIX*mail_aliases.account_id=?';
```
Comment (ChristophWurst) [inline lib/db/aliasmapper.php]: @Gomez fix it!

Comment (Gomez) [inline lib/db/aliasmapper.php]: Done!

**Comment by ChristophWurst** [comment]
:+1:

**Pull request merged**